docs/release-taxonomy.md

```diff
@@ -0,0 +1,33 @@
+[//]: # (SPDX-License-Identifier: CC-BY-4.0)
+
+**Hyperledger Governing Process document - do not modify without first getting approval of the Technical Steering Committee**
```

Comment:
As above.. I think this line is a wiki artifact and can be struck in the move to git change control.